opm/common/utility/CommunicationUtils.hpp

```diff
@@ -0,0 +1,63 @@
+/*
+  Copyright 2020,2021 OPM-OP AS
+
+  This file is part of the Open Porous Media Project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#ifndef OPM_COMMUNICATION_UTILS_HPP
+#define OPM_COMMUNICATION_UTILS_HPP
+
+#include<vector>
+#include<numeric>
+
+namespace Opm
+{
+template<class T, class A, class C>
+std::tuple<std::vector<T, A>,
```

Comment:
Since only two values are returned I guess this could use `std::pair` instead of `std::tuple`?

Reply:
My thought was that it does not matters, as it will automatically be converted to a pair if the user uses that. But testing reveals that this is not the case. Will change.